ext/bigdecimal/bigdecimal.c

```diff
@@ -2043,14 +2043,14 @@ BigDecimal_divremain(VALUE self, VALUE r, Real **dv, Real **rv)
 	b = GetVpValue(r, 0);
     }
 
-    if (!b) return DoSomeOne(self, r, rb_intern("remainder"));
+    if (!b) return Qfalse;
```

Comment:
I don't think using `Qfalse` and `Qtrue` instead of `false` and `true`, but this pattern is frequently used in `bigdecimal.c` even if the function name starts with `BigDecimal_`.
But this pattern is used in `is_zero`, `is_one` and `BigDecimal_DoDivmod`.
I think it's better to refactor in a separate pull request